graphql/util.go

```diff
@@ -598,7 +597,7 @@ type VersionModifications struct {
 	TaskIds           []string                  `json:"task_ids"` // deprecated
 }
 
-func ModifyVersion(version model.Version, user user.DBUser, proj *model.ProjectRef, modifications VersionModifications) (int, error) {
+func ModifyVersion(version model.Version, user user.DBUser, modifications VersionModifications) (int, error) {
```

Comment:
Does modifying this functions signature require you to run 'make gql' now?

Reply:
No because this is a helper function, and not specified in any schema.

Reply:
make gql should be run if either schema.graphql, resolvers.go, or, I believe gqlgen.yml are touched.